docs/import-from-whitehall.md

```diff
@@ -127,3 +127,10 @@ To unlock a document you will need the Whitehall document ID.  Type the followin
 require "gds_api/whitehall_export"
 GdsApi.whitehall_export.unlock_document(1234)
 ```
+
+To unlock all locked documents (useful when testing and comparing documents) type the following into a Content Publisher Rails console:
```

Comment:
```suggestion
To unlock all documents you have imported (useful when testing and comparing documents) type the following into a Content Publisher Rails console:
```
Minor change just to ensure there's no expectation this will unlock all documents (even those locked by someone else).

Reply:
@brucebolt Thanks Bruce.  Fixed.